master/getting-started/docker/installation/manual.md

```diff
@@ -13,7 +13,8 @@ Calico runs as a Docker container on each host. The `calicoctl` command line too
    sudo chmod +x calicoctl
    ```
 
-2. Launch `calico/node`:
+2. Setup [calicoctl - etcd datastore]({{site.baseurl}}/{{page.version}}/reference/calicoctl/setup/etcdv2).
```

Comment:
Could you tweak the text here to say "Configure calicoctl with access to your etcd cluster" or something like that to make it a bit more clear what this step is?